### DISCUSSION TITLE
PIXI.autoDetectRenderer() does not work for renderTexture.

### DISCUSSION THREAD

**Discussion by a-c-sreedhar-reddy** [pixijs/pixijs#7724] [Q&A]
```js

var app = new PIXI.Application({
    width: window.innerWidth,
    height: window.innerHeight,
    backgroundColor: 0x2c3e50
});
document.body.appendChild(app.view);

app.loader.add('bunny', 'https://image.shutterstock.com/image-illustration/green-screen-looping-animated-background-260nw-1702327750.jpg')
    .load(startup);

function startup()
{
var bunny = new PIXI.Sprite(app.loader.resources.bunny.texture);

const renderTexture = PIXI.RenderTexture.create({ height:bunny.texture.height,width:bunny.texture.width})

//This does not work
let renderer = PIXI.autoDetectRenderer();
renderer.render(bunny,{renderTexture});


//This works
// app.renderer.render(bunny,{renderTexture});

app.stage.addChild(new PIXI.Sprite(renderTexture));
}
```

### Environment

- **`pixi.js` version**: v6.1.2
- **Browser & Version**: Brave
- **OS & Version**: MacOS 11.2.3 (20D91)
- **Running Example**:https://www.pixiplayground.com/#/edit/8-Fb5LoJ8h4NpJOu6qCmT

**Comment by eXponenta** [comment]
You CANNOT share render texture between context.

When you create new renderer - you request new canvas and new WebGL context.
ALL GL RESOURCES (buffers, textures) scoped in CANVAS CONTEXT.

You MUST use RenderTexture in same renderer what create it, or use EXTRACT for grabbing render texture results into image/canvas/blob.

https://www.pixiplayground.com/#/edit/sisPjHHX6VWqZY1pithYe
